pkg/controller/miganalytic/miganalytics_controller.go

```diff
@@ -108,9 +108,9 @@ type ReconcileMigAnalytic struct {
 // MigAnalyticPersistentVolumeDetails defines extended properties of a volume discovered by MigAnalytic
 type MigAnalyticPersistentVolumeDetails struct {
 	Name                string
-	RequestedCapacity   *resource.Quantity
+	RequestedCapacity   resource.Quantity
```

Comment:
Sorry, if this is a small change, but is there a distinction between no RequestedCapacity and `0` RequestedCapacity? Could there already be a stored version on a cluster with nil here and would that cause issues on upgrade?

Reply:
@shawn-hurley valid question. AFAIK, that shouldn't cause the problem, the new struct will default to `0` values. That case will be handled further in the VolumeAdjustment logic

Reply:
Wouldn't you have a situation like this? because we don't have omit-empty?

https://play.golang.org/p/RQQFv9_mfS-

Reply:
@shawn-hurley I see your point now. This is a new change we introduced specifically for PV resize stuff in an earlier API PR. So we don't have this struct present on any MigAnalytics in previous versions. MigAnalytic controller fills this when it finds the PVs. So I think we wont come across scenario you described